content/kubeone/master/architecture/operating-system-manager/usage/_index.en.md

```diff
@@ -30,6 +30,56 @@ should be rotated manually, for them to consume configurations generated by OSM.
 
 To perform this rotation please follow the guide at [Rolling Restart MachineDeploments][rolling-restart].
 
+OSM doesn't support using `cloud-init` as provisioning utility for flatcar machines on AWS cloud provider. Flatcar use [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) instead of the upstream [cloud-init](https://cloud-init.io/). [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) is deprecated and its usage is not recommended in production environments.
+
+With OSM enabled, for new MachineDeployments the provisioningUtility will be defaulted to `ignition`, if `cloud-init` is not specified explicitly.
+
+To migrate existing MachineDeployments, make sure that you either leave the provisioningUtility field blank or set it to `ignition` explicitly.
```

Comment:
It's important to mention that you have to upgrade to latest MC for this to work, otherwise it's going to default to `cloud-init` again.

Reply:
This is the documentation for KubeOne 1.5 so it'll have the latest MC anyways no?!